doubleelim.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" href="css/w3.css">
+<head>
+    <meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TGP Knockout Calculator</title>
+</head>
+
+<body>
+	<div w3-include-html="menu.html"></div>
+	
+	<div class="w3-container w3-row" style="margin-top: 64px;">
+		<input type="number" id="playerCount" name="playerCount" min="3" max="1000" value="8" />
+		<label for="playerCount">contestants in a double elimination bracket.</label>
+	</div>
+	
+	<div class="w3-container w3-row" style="margin-top: 16px;">
+		<label for="winnersCommonFormats">Winner's Bracket Common Formats:</label>
+		<select id="winnersCommonFormats" onchange="fillWinnersBracket()">
+			<option value="1">Best-of-1</option>
+			<option value="2" selected>Best-of-3</option>
+			<option value="3">Best-of-5</option>
+			<option value="4">Best-of-7</option>
+		</select>
+	</div>
+	
+	<div class="w3-container w3-row" style="margin-top: 16px;">
+		<label for="losersCommonFormats">Loser's Bracket Common Formats:</label>
+		<select id="losersCommonFormats" onchange="fillLosersBracket()">
+			<option value="1" selected>Best-of-1</option>
+			<option value="2">Best-of-3</option>
+			<option value="3">Best-of-5</option>
+			<option value="4">Best-of-7</option>
+		</select>
+	</div>
+	
+	<div class="w3-row w3-container" style="margin-top: 16px;">
+		<div class="w3-col s12 m12 l2"><b>Winner's bracket match expected games</b></div>
+		<div class="w3-col s12 m12 l4">
+			<input type="number" id="evw" name="evw" min="1" max="50" value="2.5" />
```

Comment:
You can set up a step by 0.5; let's do that, considering the default here.  (it's currently at its default, which is 1.0)

(repeat this for the Loser's bracket)

Reply:
Done. I also normalized spacing in both files and I added a "None - Single Elimination" option for the loser's bracket length.